docs/picontrol.md

```diff
@@ -0,0 +1,77 @@
+
+- Raspberry Pi Model 3b+ (but any Raspberry Pi with Wifi capability should work)
+
+- [USB NFC readers](https://www.amazon.com/gp/product/B07Q5KXM6J/ref=ppx_yo_dt_b_asin_title_o06_s00?ie=UTF8&psc=1). These emulate a keyboard -- when an NFC badge is scanned onto them, they 'type' its ID into whatever they're connected to.
+  - **PLEASE** make note of the model: if that link isn't dead, you'll see it links to a '14H' reader. '14H' (or '14D') is required to get the specificity necessary to distinguish between the NFC badge IDs: these readers read the most significant bit first, and since the card IDs are assigned sequentially this means that most of them have the same first 8-10 hex digits. You are in for much pain if you do not purchase the correct readers.
+- Microusb battery pack. Best to get flat ones.
+- *Optionally*: LEDs
+
+### Hardware: Future Considerations
+
+- Raspberry Pi's are certainly overkill for this application. Since the boxes only need to make a single web request, a simpler IoT board (eg. ESP8266 or ESP32) would work just as well, consume far less power, and cost an order of magnitude less.
+  - As a corollary, "PiControl" is probably a bad name for this system as pretty much all of the logic is hardware-agnostic. Even the small Python script run on the Pi could probably run on a much simpler microcontroller (with a couple modifications) through Micropython.
+- The keyboard emulation NFC readers were used because Katz (the author of this doc) had never touched hardware in his life and thus couldn't figure out how to use the [an actual breakout board](https://www.adafruit.com/product/364). In the first week of CS107e, he realized that he was connecting the board's TX and RX pins to the Pi's TX and RX pins (respectively). He has now learned that TX == "transmit" and RX == "receive" and thus TX on one board should be connected to RX on the other and vice versa. If you ever see Katz, feel free to chastise him for his foolishness.
+
+## Software
+
+### On the Pi
+
+The script run on the Pi can be found at https://github.com/MYKatz/PiControl. If you're using the aforementioned USB NFC readers, you should be running 'read_keyboard.py'.
+
+The script is very simple: upon receiving input from the NFC reader (a card's ID), it sends that to a specified API endpoint.
+
+#### OS and dependencies
+
+We used a standard install of Raspbian. You will need to configure the Pi to connect to Wifi -- there are many guides online to do this, such as [this one](https://raspberrypihq.com/how-to-connect-your-raspberry-pi-to-wifi/).
+
+An installation of Python 3.7 should include all of the necessary dependencies except for [Requests](https://requests.readthedocs.io/en/master/). A 'pip3 install requests' should work, provided you're connected to the internet.
+
```

Comment:
Can you add a section here about changing the default password for the raspberry pi's?